test/src/test/kotlin/graphql/nadel/tests/transforms/RemoveFieldTestTransform.kt

```diff
@@ -78,11 +77,11 @@ class RemoveFieldTestTransform : NadelTransform<GraphQLError> {
         state: GraphQLError,
         nodes: JsonNodes,
     ): List<NadelResultInstruction> {
-        val parentNodes = JsonNodeExtractor.getNodesAt(
-            data = result.data,
-            queryPath = underlyingParentField?.queryPath ?: NadelQueryPath.root,
-            flatten = true,
-        )
+        val parentNodes = JsonNodes(result.data)
```

Comment:
This should use the `nodes` param instead of creating a new one.

I mean it's test code so it doesn't matter but in reality we want to share one `JsonNodes` because it caches some JSON traversals.